examples/bat_gen/runBatFit.cxx

```diff
@@ -36,8 +37,10 @@ int main()
     yap::plainLogs(el::Level::Info);
 
     // open file
-    std::string model_name = "D3PI";
-    // std::string model_name = "DKKPI";
+    // std::string model_name = "D3PI";
+    //std::string model_name = "DKKPI";
+    std::string model_name = "D4PI";
```

Comment:
add 4pi as a commented out option;
or better yet: add switches to determine. `if (true) { ... } else { ... }`